data/dwarfplanets.ssc

```diff
@@ -704,25 +704,34 @@ ReferencePoint "Orcus-Vanth" "Sol"
 	}
 	EllipticalOrbit
 	{
-		Epoch	2454000.0  # 2006 Sept 21 12:00
-		Period	9.3072
-		SemiMajorAxis	940
-		Eccentricity	0.021
-		Inclination	100.2
-		AscendingNode	87.2
-		ArgOfPericenter	305.6
-		MeanAnomaly	321.4
+		Epoch	2454000  # 2006 Sept 21 12:00
+		Period	9.539154
+		SemiMajorAxis	1241.4  # semi-major axis: 8999.8 km, mass ratio: 0.16 +/- 0.02
+		Eccentricity	0.00091
+		Inclination	105.03
+		MeanLongitude	8.52
+		AscendingNode	53.49
+		LongOfPericenter	148
+	}
+	BodyFrame
+	{
+		EquatorJ2000	{ Center "Sol/Orcus-Vanth" }
 	}
 	UniformRotation  # to match mutual orbit
 	{
-		Inclination	98.2
-		AscendingNode	83.3
+		Epoch	2454000  # 2006 Sept 21 12:00
+		Inclination	105.03
+		AscendingNode	53.49
+		MeridianAngle	135.03
 	}
 	LunarLambert	0.5
 	GeomAlbedo	0.231
 	InfoURL	"https://en.wikipedia.org/wiki/90482_Orcus"
 }
 
+# Orbit from Grundy et al. (2019), Icarus 334, 62-78
+# "Mutual orbit orientations of transneptunian binaries"
+# https://ui.adsabs.harvard.edu/abs/2019Icar..334...62G/abstract
```

Comment:
May I suggest that this freely-accessible link be used instead? http://www2.lowell.edu/users/grundy/abstracts/2019.TNB_orbits.html
(same applies to the other instances where this paper is cited)

Reply:
Sounds good; suggestion added and will be in the next commit.